### PR TITLE
OAuth Design QA C-534 C-533 C-532

### PR DESCRIPTION
**Pull request by nicoback2**
### Description
-Make sign out button hover state more prominent
-Fix sign out button text clipping on Safari
-Fix email loading icon misaligned on Safari

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--qa-oauth

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--qa-oauth

File: packages/web/src/pages/oauth-login-page/OAuthLoginPage.tsx [hunk 1]
```diff
@@ -353,11 +351,13 @@ export const OAuthLoginPage = () => {
                     [styles.permissionTextExtraLight]: !userEmail
                   })}
                 >
-                  {userEmail || (
+                  {userEmail == null ? (
```
Comment (sliptype) [inline packages/web/src/pages/oauth-login-page/OAuthLoginPage.tsx:354]: Is there a case where `userEmail` is undefined? Should we be using `===` here

**Review by sliptype — APPROVED** [review]

File: packages/web/src/pages/oauth-login-page/OAuthLoginPage.tsx (hunk 1 above)
Comment (nicoback2) [reply packages/web/src/pages/oauth-login-page/OAuthLoginPage.tsx:354]: I always use == null by convention, but === would work here too!

**Pull request merged**